how-it-works/tipping.md

```diff
@@ -1,3 +1,11 @@
-# Tipping
+# 💡 Tipping System: Acknowledging and Rewarding Your Contributions 🌟
 
-As a open source maintainer, you have the opportunity to reward your community outside of bounties.
```

Comment:
https://docs.github.com/en/get-started/learning-about-github/access-permissions-on-github

Reply:
who can i fix this issuse